test/unit/cart.js

```diff
@@ -91,6 +91,19 @@ describe('Cart class', () => {
     });
   });
 
+  describe('fetchData()', () => {
+    it('calls createCart on client if localStorage invalid', () => {
+      localStorage.setItem('checkoutId', 1);
+      const createCheckout = sinon.stub(cart.props.client.checkout, 'create').returns(Promise.resolve({id: 12345, lineItems: []}));
```

Comment:
This isn't testing what you think it's checking. You should be stubbing `checkout.fetch` to return a `Promise.reject`, then ensure that `createCheckout` is called. We want to see that when fetch fails, create is called.